### PR TITLE
[17.0][MIG] partner_contact_access_link: Migration to 17.0

### PR DESCRIPTION
**Pull request by Tisho99**
Supersedes #1649

Cleared commit history 

T-6069

### CODE REVIEW

**Comment by Tisho99** [comment]
@ValentinVinagre @HaraldPanten

**Comment by Tisho99** [comment]
@CRogos 

I have cleared commit history as you suggested in #1649

File: base_partner_sequence/README.rst
Comment (CRogos) [inline base_partner_sequence/README.rst:1]: Can you remove the changes form base_partner_sequence

**Review by CRogos — CHANGES_REQUESTED** [review]

File: base_partner_sequence/README.rst
Comment (Tisho99) [reply base_partner_sequence/README.rst:1]: Done!

**Review by CRogos — APPROVED** [review]
Code + Function LGTM

**Review by HaraldPanten — APPROVED** [review]
LGTM

**Pull request merged**